core/kazoo_tasks/src/kz_task_worker.erl

```diff
@@ -0,0 +1,137 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2016, 2600Hz INC
+%%% @doc
+%%% Run tasks scheduled by parent.
+%%% @end
+%%% @contributors
+%%%   Pierre Fenoll
+%%%-------------------------------------------------------------------
+-module(kz_task_worker).
+
+%% API
+-export([start/6]).
+
+-include("tasks.hrl").
+
+-record(state, { task_id :: kz_tasks:task_id()
+               , module :: module()
+               , function :: atom()
+               , fassoc :: kz_csv:fassoc()
+               , extra_args :: kz_proplist()
+               , total_succeeded = 0 :: non_neg_integer()
+               }).
+
+-define(IN, 'csv_in').
+
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec start(kz_tasks:task_id()
+           ,module()
+           ,atom()
+           ,kz_proplist()
+           ,ne_binaries()
+           ,ne_binary()
+           ) -> any().
+start(TaskId, Module, Function, ExtraArgs, OrderedFields, AName) ->
+    _ = kz_util:put_callid(TaskId),
+    case init(TaskId, Module, Function, ExtraArgs, OrderedFields, AName) of
+        {'ok', State} ->
+            lager:debug("worker for ~s started", [TaskId]),
+            loop(State);
+        {'error', _R} ->
+            kz_tasks:worker_finished(TaskId, 0),
+            lager:debug("worker exiting now: ~p", [_R])
+    end.
+
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+%% @private
+init(TaskId, Module, Function, ExtraArgs, OrderedFields, AName) ->
+    case kz_util:try_load_module(Module) == Module andalso
```

Comment:
`andalso` should be on the new line, preceding the conditional. I added [code_checks.bash](https://github.com/2600hz/kazoo/blob/master/scripts/code_checks.bash) to help identify these types of formatting cases. Not sure if Elvis could manage it but was easy enough to script.